plugins/woocommerce/tests/e2e-pw/tests/merchant/page-loads.spec.js

```diff
@@ -66,6 +66,21 @@ for ( const currentPage of wcPages ) {
 						await page.waitForLoadState( 'networkidle' );
 						await page.goto( 'wp-admin/admin.php?page=wc-admin' );
 					}
+
+					// deal with cases where the 'Coupons' legacy menu had already been removed.
+					if ( currentPage.subpages[ i ].name === 'Coupons' ) {
```

Comment:
In our nightly smoke test site, the legacy WooCommerce > Coupons menu had already been removed, since there's an option for the user to remove it. This condition will skip the test in such cases.